pyproject.toml

```diff
@@ -0,0 +1,89 @@
+[project]
+name = "python-library-template"
+version = "0.1.0"
+description = "Template python repo with our proven approaches."
+authors = [
+    "Adam Frank <afrank@mozilla.com",
+    "Bryan Sieber <bsieber@mozilla.com>",
+    "Chris Valaas <cvalaas@mozilla.com",
+    "Jorge Spiropulo <jspiropulo@mozilla.com"
+]
+readme = "README.md"
+python = "^3.7"
```

Comment:
@bsieber-mozilla I'm already using python 3.8 locally, can we specify ">=3.4" unless anyone disagrees?

Reply:
Oops, I thought this was a >=3.7! Want >=3.7 or 3.4?

Reply:
I've been using >=3.4 but I'd also be totally fine with >=3.7 at this point.

Reply:
Swapped it to >=!